### ISSUE TITLE
ram: Code:"EntityNotExist.User", 

### ISSUE THREAD

**Issue by bittopaz** [alibaba/terraform-provider#679]
When upgrade the provider from 1.9.4 to 1.15.0, we are seeing the errors as below, 

```
* alicloud_ram_user_policy_attachment.dev_default: 1 error(s) occurred:

* alicloud_ram_user_policy_attachment.dev_default: alicloud_ram_user_policy_attachment.dev_default: Get list policies for user got an error: &common.Error{ErrorResponse:common.ErrorResponse{Response:common.Response{RequestId:"FB19E56C-2BEB-4BFC-B8EC-ACA3377034A6"}, HostId:"ram.aliyuncs.com", Code:"EntityNotExist.User", Message:"The user does not exist."}, StatusCode:404}
```

We have to rolled it back to 1.9.4

**Comment by bittopaz** [comment]
@xiaozhu36  can you please take a look at this issue? 

Currently this is blocking us from upgrading to 1.15.0?

**Comment by bittopaz** [comment]
Confirmed, fixed by #712.
